frontend/src/components/Song.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useRef, useState } from "react";
+import { Text, Text3D } from "@react-three/drei";
+import { animated, useSpring } from "@react-spring/three";
+import { v4 as uuidv4 } from "uuid";
+
+import data from "../utils";
+const toneArmOnSoundeffect = new Audio("/soundeffects/tonearm_on_sound.mp3");
+toneArmOnSoundeffect.volume = 0.4;
+const vinylSoundeffect = new Audio("/soundeffects/vinyl_soundeffect.mp3");
+vinylSoundeffect.volume = 1;
+vinylSoundeffect.loop = true;
+
+export default function Song({
+  playing,
+  setPlaying,
+  toneArmFinished,
+  soundOn,
+  songIndex,
+}) {
+  const x = -3.2;
+  const y = 1.4;
+  const z = 4;
```

Comment:
Possible to rename to better reflect what these coordinate points are used for,

I see it used in animated text so maybe 

`const xAnimatedText` 
`const yAnimatedText`
`const zAnimatedText`